patroni/postgresql/slots.py

```diff
@@ -48,6 +48,20 @@ def sync_replication_slots(self, cluster):
 
                 # drop old replication slots which are not presented in desired slots
                 for name in set(self._replication_slots) - set(slots):
+                    ignore_slot = False
+                    for ignore_slots_matcher in cluster.config.ignore_slots_matchers:
+                        slot = self._replication_slots[name]
+                        if (
+                                (ignore_slots_matcher.get("name") is None or ignore_slots_matcher["name"] == name)
+                                and (ignore_slots_matcher.get("database") is None or ignore_slots_matcher["database"] == slot["database"])
+                                and (ignore_slots_matcher.get("plugin") is None or ignore_slots_matcher["plugin"] == slot["plugin"])
+                                and (ignore_slots_matcher.get("type") is None or ignore_slots_matcher["type"] == slot["type"])
+                            ):
```

Comment:
Flake8 (on travis) complains about these lines:
```
patroni/postgresql/slots.py:56:121: E501 line too long (138 > 120 characters)
patroni/postgresql/slots.py:57:121: E501 line too long (132 > 120 characters)
patroni/postgresql/slots.py:58:121: E501 line too long (126 > 120 characters)
patroni/postgresql/slots.py:59:29: E125 continuation line with same indent as next logical line
```

You can also check it locally as:
```
python3 setup.py flake8
```

Reply:
Fixing.